src/core/carouselList/CarouselList.tsx

```diff
@@ -0,0 +1,137 @@
+import React, { useEffect, useState } from 'react'
+// eslint-disable-next-line import/no-extraneous-dependencies
+import classNames from 'classnames'
+import { IconAngleLeft, IconAngleRight } from 'hds-react'
+
+import styles from './carouselList.module.scss'
+import splitArrayIntoChunksOfLen from './utils/utils'
+
+export type CarouselListProps<T> = {
+  children: React.ReactElement<T>[]
+  itemsDesktop?: 1 | 2 | 3 | 4 | 5
+  itemsMobile?: 1 | 2
+}
+
+export default function CarouselList({
```

Comment:
In Bootstrap it's just a "carousel" https://getbootstrap.com/docs/4.0/components/carousel/. CarouselList is OK also, but I would consider the Carousel, because it's just smaller and I think it cannot be mixed to anything else. And there cannot be any other kind of carousels, e.g "carousel grid" or "carousel paragraph".

Reply:
I can rename, it was just to be consistent as we had the "grid list". but if we rename one, then other should be fixed as well